core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -197,7 +197,7 @@ private static void setupLogging(Settings settings, Environment environment) {
 
     private static Tuple<Settings, Environment> initialSettings(boolean foreground) {
         Terminal terminal = foreground ? Terminal.DEFAULT : null;
-        return InternalSettingsPreparer.prepareSettings(EMPTY_SETTINGS, true, terminal);
+        return InternalSettingsPreparer.prepareSettingsAndEnvironment(EMPTY_SETTINGS, terminal);
```

Comment:
it might be simpler to just name this `InternalSettingsPreparer.prepareEnvironment` and return only `Environment` since it has the settings already as a member then we can spare the tuple and things are less funky?